routers/api/v1/repo/issue.go

```diff
@@ -912,24 +912,29 @@ func EditIssue(ctx *context.APIContext) {
 			}
 		}
 
-		var isClosed bool
+		var closeOrReopen bool
```

Comment:
I think this variable name `closeOrReopen` is a bit weird, but I have no idea how to improve it. 🤔